public/node.html

```diff
@@ -0,0 +1,168 @@
+
+      <a href='/'>
+        <img class='logo' src="img/logo.png">
+        <h1>Learn JS Data</h1>
+        <h3>Data manipulation, munging, and processing in JavaScript</h3>
+      </a>
+    </div>
+  </header>
+
+  <div id="container">
+    <div id="background"></div>
+    
+    <ul class="sections">
+        
+        
+        
+        <li id="section-1">
+            <div class="annotation">
+              
+              <div class="pilwrap ">
+                <a class="pilcrow" href="#section-1">&#182;</a>
+              </div>
+              <h1 id="analyzing-data-with-node">Analyzing Data with Node</h1>
+<p>As mentioned in the <a href="getting_started.html">introduction</a>, this guide is mostly geared for client-side data analysis, but with a few augmentations, the same tools can be readily used server-side with <a href="https://nodejs.org/">Node</a>.</p>
+<p>And the data is too large, this might in fact be your only option if you want to use JavaScript for your data analysis. None of your users will wait 5 minutes on your site while a giant file is loaded up, no matter how cool the analysis.</p>
+<h2 id="setting-up-a-node-project">Setting up a Node Project</h2>
+<p>To get started with Node, ensure both node and <a href="https://www.npmjs.com/">npm</a>, the Node package manager, are installed and available via the command line:</p>
+<pre><code class="lang-bash"><span class="hljs-built_in">which</span> node
+<span class="hljs-comment"># /usr/local/bin/node</span>
+<span class="hljs-built_in">which</span> npm
+<span class="hljs-comment"># /usr/local/bin/npm</span>
+</code></pre>
+<p>Your paths may be different then mine, but as long as <code>which</code> returns something, you should be good to go. </p>
+<p>Create a new directory for your data analysis project. In this example, we have a directory with a sub-directory called <code>data</code> which contains our <code>animals.tsv</code> file inside.</p>
+<h2 id="installing-node-modules">Installing Node Modules</h2>
+<p>Next, we want to install our JavaScript tools, D3 and lodash. With Node, we can automate the process by using <code>npm</code>. Inside your data analysis directory run the following:</p>
+<pre><code class="lang-bash">npm install d3
+npm install lodash
+</code></pre>
+<p>You can see that npm creates a new sub-directory called <code>node_modules</code> by default, where your packages are installed. Everything is kept local, so you don’t have to worry about problems with missing or out-of-date packages. Your analysis tools for each project are ready to go.</p>
+<h2 id="requiring-modules">Requiring Modules</h2>
+<p>Now we create a separate JavaScript file to do our analysis in:</p>
+<pre><code class="lang-bash">touch analyze.js
+</code></pre>
+<p>Inside this file, we first <a href="http://openmymind.net/2012/2/3/Node-Require-and-Exports/">require</a> our external dependencies. With Node, everything is explicit. You need to indicate exactly which tools you want to use.</p>
```

Comment:
I wouldn't say that 'require' makes node more explicit. You still have to load files in client side js explicitly. You could say it is synchronous and guaranteed to finish before moving on to the next line or something. But i wasn't sure what you were trying to say.